content/en/_index.md

```diff
@@ -3,48 +3,35 @@ title: OpenTelemetry
 show_banner: true
 ---
 
-{{< blocks/cover title="" image_anchor="center" height="min" color="dark" >}}
-<div class="mx-auto">
-    <div class="col-sm-5 offset-sm-7 card bg-transparent border-0">
-      <img class="card-img-top mx-auto" src="/img/logos/opentelemetry-stacked-color.png">
-      <div class="card-body">
-        <h3 class="card-title text-left">An observability framework for cloud-native software.</h3>
-        <p class="card-text text-left"><small>OpenTelemetry is a collection of tools, APIs, and SDKs. You can use it to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior.</small></p>
-      </div>
-    </div>
-  <br/>
-  <div class="card-group">
-    <div class="card bg-primary">
-      <div class="card-body">
-        <h4 class="card-title">Broad Language Support</h4>
-        <p>Java | C# | Go | JavaScript | Python | Rust | C++ | Erlang/Elixir</p>
-      </div>
-      <div class="card-footer bg-transparent">
-        <a class="btn btn-lg btn-primary" href="{{< relref "/docs" >}}">
-          Get Started <i class="fas fa-arrow-alt-circle-right ml-2"></i>
-        </a>
-      </div>
-    </div>
-    <div class="card bg-secondary">
-      <div class="card-body">
-        <h4 class="card-title">Integrates With Popular Frameworks and Libraries</h4>
-        <p>MySQL | Redis | Django | Kafka | Jetty | Akka | RabbitMQ | Spring | Quarkus | Flask | net/http | gorilla/mux | WSGI | JDBC | PostgreSQL</p>
-      </div>
-      <div class="card-footer bg-transparent">
-        <a class="btn btn-lg btn-secondary" href="https://github.com/open-telemetry">
-          Find Projects <i class="fab fa-github ml-2"></i>
-        </a>
-      </div>
-    </div>
-  </div>
-  <div class="mx-auto mt-5">
-    {{< blocks/link-down color="primary" >}}
-  </div>
+{{< blocks/cover image_anchor="top" height="max" color="primary" >}}
+<img src="/img/logos/opentelemetry-horizontal-color.svg" class="otel-logo" />
+<h2>An observability framework for cloud-native software</h2>
+<a
+	class="btn btn-lg btn-primary font-weight-bold mt-5 my-4"
+	href="/docs/concepts/"
+>
+Learn more
+</a>
+
+<div class="h3 mt-2">Get started!</div>
+
+<div class="l-get-started-buttons">
+
+- [Collector]({{< relref "/docs/collector/getting-started" >}})
+- [Go]({{< relref "/docs/go/getting-started" >}})
+- [.Net]({{< relref "/docs/net/getting-started" >}})
+- [Javascript]({{< relref "/docs/js" >}})
```

Comment:
**Getting started** button order is based on analytics.

Reply:
nit (and double check me here) but should it be `.NET` and `JavaScript`?

Reply:
Yes, and yes! Thanks for catching that.

Note that the title of https://opentelemetry.io/docs/js/ is "Javascript". I'll fix that later.